package.json

```
@@ -41,13 +41,14 @@
   },
   "peerDependencies": {
     "intl": "^1.2.5",
-    "react-intl": "^2.8.0"
+    "react-intl": "^2.8.0 || ^5.0.0"
   },
   "devDependencies": {
     "@cerner/eslint-config-terra": "^4.0.0",
     "eslint": "^6.1.0",
     "jest": "^24.7.1",
-    "memory-fs": "^0.4.1"
+    "memory-fs": "^0.4.1",
+    "react-intl": ">2.8.0"
```

lets match these up. Not sure if theres a case where the dev dependency would install v6 (if there was one)?
```suggestion
    "react-intl": "^2.8.0 || ^5.0.0"
```